src-test/com/zone5cloud/retrofit/core/TestUsersAPI.java

```diff
@@ -174,4 +180,37 @@ public void testUserPreferences() {
 		assertTrue(userApi.setPreferences(p).blockingFirst().body());
 		assertEquals(UnitMeasurement.metric, userApi.getPreferences(userId).blockingFirst().body().getMetric());
 	}
+
+	@Test
+	public void testPasswordComplexity(){
+		String passwordFormat = "^(?=.[\\d])(?=.[a-z])(?=.[A-Z])(?=.[a-zA-Z]).{8,}$";
+		UserAPI userAPI = mock(UserAPI.class);
+		when(userAPI.passwordComplexity()).thenReturn(Observable.just(Response.success(passwordFormat)));
+		Response<String> response = userAPI.passwordComplexity().blockingFirst();
+		assertEquals(response.body(),passwordFormat);
+	}
+
+	@Test
+	public void testReconfirmEmail(){
+		String email = "sometest@email.com";
+		UserAPI userAPI = mock(UserAPI.class);
+		when(userAPI.reconfirm(email)).thenReturn(Observable.just(Response.success(200,null)));
+		Response<Void> response =  userAPI.reconfirm(email).blockingFirst();
+		assertEquals(200,response.code());
+	}
+
+	@Test
+	public void testReconfirm() {
+		login();
+		String email = userApi.me().blockingFirst().body().getEmail();
+		Response<Void> response = userApi.reconfirm(email).blockingFirst();
+		assertEquals(200,response.code());
+	}
+
+	@Test
+	public void testPasswordComplexityApi() {
+		login();
+		Response<String> response = userApi.passwordComplexity().blockingFirst();
+		assertNotNull(response.body());
+	}
```

Comment:
what is the response body? Does it look sane to you? Is it a regex string? I had issues in the iOS library where this value was being garbled - so just being pre cautious and want to make sure it is working as expected.